packages/block-library/src/image/image.js

```diff
@@ -689,8 +697,8 @@ export default function Image( {
 					onResizeStop();
 					setAttributes( {
 						width: elt.offsetWidth,
-						height: elt.offsetHeight,
-						aspectRatio: undefined,
+						height: 'auto',
+						aspectRatio: aspectRatio ? aspectRatio : undefined,
```

Comment:
I'm not sure why this wasn't set before?

Reply:
The drag handles set both the `width` and the `height` so the `aspectRatio` needed to be cleared since it isn't valid with both `width` and `height`.